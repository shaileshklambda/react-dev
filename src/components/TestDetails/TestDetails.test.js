import React from "react";
import renderer from "react-test-renderer";
import Testdetails from ".";

it("Testdetails: default", () => {
  const component = renderer.create(<Testdetails />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
