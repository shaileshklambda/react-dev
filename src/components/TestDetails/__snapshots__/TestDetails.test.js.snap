// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testdetails: default 1`] = `
<div
  className="col-md-12 col-xs-12"
>
  <div>
    <div
      className="row"
    >
      <div
        className="col-md-7 col-xs-7"
      >
        <div
          className="row"
        >
          <div
            className="headingParent col-md-12 col-xs-12"
          >
            <div
              className="col-md-12 col-xs-12"
            >
              <span
                className="testHeading"
              >
                Falcon/Realtime Browser
              </span>
              <span
                className="highlightedText"
              >
                 
                 FS-006S
              </span>
            </div>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-md-12 col-xs-12"
          >
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    Tester: 
                  </strong>
                  Gragnama
                </p>
              </span>
            </div>
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    CI Tool: 
                  </strong>
                   
                  Jenkins
                </p>
              </span>
            </div>
          </div>
          <div
            className="col-md-12 col-xs-12"
          >
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    Local Tunnel: 
                  </strong>
                   
                  Null
                </p>
              </span>
            </div>
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    No. of Issues: 
                  </strong>
                   
                  20
                </p>
              </span>
            </div>
          </div>
          <div
            className="col-md-12 col-xs-12"
          >
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    Start Time: 
                  </strong>
                   
                  16:00
                </p>
              </span>
            </div>
            <div
              className="col-md-6 col-xs-6"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    End Time: 
                  </strong>
                   
                  20:00
                </p>
              </span>
            </div>
          </div>
          <div
            className="col-md-12 col-xs-12"
          >
            <div
              className="col-md-12 col-xs-12"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    Base Url: 
                  </strong>
                   
                  https://www.lambdatest.com
                </p>
              </span>
            </div>
          </div>
          <div
            className="col-md-12 col-xs-12"
          >
            <div
              className="col-md-12 col-xs-12"
            >
              <span
                className="innerContent"
              >
                <p>
                  <strong>
                    Environment: 
                  </strong>
                   
                  <img
                    className=""
                    src="icons/windows.png"
                  />
                   
                  10
                   
                  |
                   
                  <img
                    className=""
                    src="icons/google_chrome.png"
                  />
                   
                  69.0
                   
                  |
                   
                  <img
                    className=""
                    src="icons/computer.png"
                  />
                   
                  1900 x 1080
                </p>
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-md-4 col-xs-4"
      >
        <div>
          <div
            data-vjs-player={true}
          >
            <video
              className="video-js vjs-big-play-centered"
              poster=""
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
